build/cmake/functions.cmake

```diff
@@ -342,22 +351,23 @@ function(wx_set_target_properties target_name is_base)
     if(wxBUILD_SHARED)
         string(TOUPPER ${target_name_short} target_name_upper)
         if(target_name_short STREQUAL "mono")
-            target_compile_definitions(${target_name} PRIVATE DLL_EXPORTS WXMAKINGDLL)
-        else()
-            target_compile_definitions(${target_name} PRIVATE DLL_EXPORTS WXMAKINGDLL_${target_name_upper})
-        endif()
-        if(NOT target_name_short STREQUAL "base")
-            target_compile_definitions(${target_name} PRIVATE WXUSINGDLL)
+            target_compile_definitions(${target_name} PRIVATE WXMAKINGDLL)
```

Comment:
Maybe
```suggestion
            target_compile_definitions(${target_name} PRIVATE $<BUILD_INTERFACE:WXMAKINGDLL> PUBLIC $<INSTALL_INTERFACE:WXUSINGDLL>)
```

Reply:
This `target_compile_definitions` is conditional to only the `mono` project, so it cannot be combined with the one below that applies to all projects.